### DISCUSSION TITLE
Enable verbose

### DISCUSSION THREAD

**Discussion by stiny861** [tteck/Proxmox#2164] [Q&A] [answered]
I am very new to Proxmox and have been using a bunch of these setup scripts. I hit an issue with one of them and was going to submit a bug report, but saw the note about verbose logging first to get an idea of what is wrong. Question is, how do I do that? I have looked everywhere i can think, as well as just the standard throw either -verbose or --verbose at the end of the script in the shell, but that is not working. Can anyone throw me a bone and tell me how to enable verbose mode when running the script? 

On a side note, since that is specifically called out in the bug report section, a suggestion to tteck would be to throw instructions on how to enable verbose logging if it is a requirement for submitting a bug report.

Thanks all

**Comment by tteck** [comment, marked as answer]
Use "Advanced" settings to enable Verbose mode

**Reply by stiny861** [reply]
Doh. Man I feel dumb. Never thought to look in advanced.
